Firmware/Marlin_main.cpp

```diff
@@ -8763,6 +8763,106 @@ SERIAL_PROTOCOLPGM("\n\n");
 	}
 	break;
 
+    /*!
+    ### M704 - Load to MMU
+    #### Usage
+
+        M704 [ P ]
+
+    #### Parameters
+    - `P` - n index of slot (zero based, so 0-4 like T0 and T4)
+    */
+    case 704:
+    {
+        uint8_t mmuSlotIndex = 0xffU;
```

Comment:
it looks like this part is common to all M704, 5, 6, it could be extracted to a common function later to save space

Reply:
I don't see any easy way to create a function pointer. C++ syntax seems overly complicated :(

Reply:
Here is a less clean solution but it saves 36 bytes of flash
```c
static void gcodes_M704_M705_M706(uint16_t gcode)
{
    uint8_t mmuSlotIndex = 0xffU;
    if (MMU2::mmu2.Enabled() && code_seen('P'))
    {
        mmuSlotIndex = code_value_uint8();
        if (mmuSlotIndex < MMU_FILAMENT_COUNT) {
            switch (gcode)
            {
            case 704:
                MMU2::mmu2.load_filament(mmuSlotIndex);
                break;
            case 705:
                MMU2::mmu2.eject_filament(mmuSlotIndex, false);
                break;
            case 706:
                MMU2::mmu2.cut_filament(mmuSlotIndex);
                break;
            default:
                break;
            }
        }
    }
}
```